### PR TITLE
Technical review feedback integration from jsrain

### PR DESCRIPTION
**Pull request by Loquacity**

### CODE REVIEW

File: adoc/retail_chap_components.adoc [hunk 1]
```diff
@@ -47,20 +47,23 @@ In this configuration, it may or may not provide DHCP and DNS services.
...
-Point-of-sale (PoS) terminals can come in many different formats, such as cashier terminals, digital scales, or cash drawers.
+Point-of-sale (PoS) terminals can come in many different formats, such as cashier terminals, digital scales, or information kiosks.
 Every terminal, however, is provided by a vendor, who set some basic information about the device in the firmware.
 {smr} accesses this vendor information in order to determine how best to work with the terminal in use.
 In most cases, different terminals will require a different operating system (OS) image to ensure they work correctly.
-For example, a cashier terminal has a full graphical display, where a digital scale might only have a very basic text display.
+For example, an information kiosk has a high-resolution touchscreen, where a cashier terminal might only have a very basic display.
 While both of these terminals require similar processing and network functionality, they will require different OS images to ensure that the different display mechanisms work correctly.
 
+Note that {smr} works well for deployments containing up to around 5,000 terminals.
```
Comment (keichwa) [inline adoc/retail_chap_components.adoc]: Wondering about number separators.  Sometimes people use these separators only for numbers >9999.

File: adoc/retail_chap_install.adoc [hunk 2]
```diff
@@ -9,7 +9,8 @@ include::entities.adoc[]
 [[retail.sect.running.requirements]]
 == Requirements
 
-Before you install {smr}, ensure your environment meets the minimum requirements:
+Before you install {smr}, ensure your environment meets the minimum requirements.
+In addition to the {susemgr} requirements listed at https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-getting-started/single-html/book.suma.getting-started/book.suma.getting-started.html#quickstart.sect.prereq.hardware and https://www.suse.com/documentation/suse-manager-3/singlehtml/suse_manager21/book_susemanager_proxyquick/book_susemanager_proxyquick.html your {smr} installation will require:
```
Comment (keichwa) [inline adoc/retail_chap_install.adoc]: It feels wrong referencing the **2.1** proxy QS.

File: adoc/retail_chap_components.adoc (hunk 1 above)
Comment (Loquacity) [reply adoc/retail_chap_components.adoc]: Hrm, good question. IBM says to omit the comma in four digit numbers unless needed 'for consistency', Chicago says to leave them in. Do we have style guidance on this one?

File: adoc/retail_chap_install.adoc (hunk 2 above)
Comment (Loquacity) [reply adoc/retail_chap_install.adoc]: Huh, good spot. Might need to check with @jsrain

File: adoc/retail_chap_components.adoc (hunk 1 above)
Comment (keichwa) [reply adoc/retail_chap_components.adoc]: > Hrm, good question. IBM says to omit the comma in four digit numbers unless needed 'for consistency', Chicago says to leave them in. Do we have style guidance on this one?

I think we don't have style guidance.  I grep'ed through our files:

```
bp_chap_systems_management.adoc:In the example above, we have assumed 500 current clients, so the soft limit is 5100, and the hard limit is 6000.
For example in the case of 5000 clients: `jabbersoftnofile5100 jabberhardnofile6000`
```

This seems to indicate that the comma in four digit numbers isn't needed 'for consistency'. ;)

Comment (Loquacity) [reply adoc/retail_chap_components.adoc]: Sure, that's good enough for me. I'll take it out :)

Comment (Loquacity) [reply adoc/retail_chap_components.adoc]: https://github.com/SUSE/doc-susemanager/pull/208

File: adoc/retail_chap_install.adoc [hunk 3]
```diff
@@ -225,11 +227,11 @@ Here is a channel overview:
 +
 *TODO*: add more info about these 3 actions if wanted; it's in Advanced Topics, Proxy QS.
 +
-*TODO*: do we have a suma_retail_proxy pattern?
+*TODO*: do we have a suma_retail_proxy pattern? According to Jiri: yes.
```
Comment (jsrain) [inline adoc/retail_chap_install.adoc]: May be a misunderstanding: We have the suma_proxy pattern (not retail specific) and we have a retail pattern for the SUMA Server (but not proxy)

File: adoc/retail_chap_install.adoc (hunk 2 above)
Comment (jsrain) [reply adoc/retail_chap_install.adoc]: @Loquacity Seems that Proxy does not have a standalone book in 3.2, but is only covered by the SUSE Manager Advanced Topics Guide. The link below should provide the needed requirements:

https://www.suse.com/documentation/suse-manager-3/3.2/susemanager-advanced-topics/single-html/book.suma.advanced.topics/book.suma.advanced.topics.html#at.manager.proxy.requirements

Comment (Loquacity) [reply adoc/retail_chap_install.adoc]: https://github.com/SUSE/doc-susemanager/pull/209

Also, @keichwa apparently we've been doing IDs and xrefs all wrong :S https://github.com/SUSE/doc-susemanager/wiki/asciidoc-getting-started#links

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Also, @keichwa apparently we've been doing IDs and xrefs all wrong :S
> https://github.com/SUSE/doc-susemanager/wiki/asciidoc-getting-started#links

yes!  xref: is brand-new tech for us (me).  I still do not understand
why we must add the title/label text within the []...

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nürnberg)

**Comment by Loquacity** [comment]
I would argue that we need the alt text for accessibility. If you're using a screen reader, when it hits a URL it takes *forever* to read it all out. If there's alt text then it's a much nicer experience.

**Pull request merged**